Ladybird/CMakeLists.txt

```diff
@@ -260,6 +162,12 @@ function(create_ladybird_bundle target_name)
             COMMAND "${CMAKE_COMMAND}" -E make_directory "${bundle_dir}/Contents/Resources"
             COMMAND "iconutil" --convert icns "${LADYBIRD_SOURCE_DIR}/Ladybird/Icons/macos/app_icon.iconset" --output "${bundle_dir}/Contents/Resources/app_icon.icns"
         )
+
+        # Note: This symlink is removed in the install commands
+        # This makes the bundle in the build directory *NOT* relocatable
+        add_custom_command(TARGET ${target_name} POST_BUILD
```

Comment:
Ugh. I don't really like this, but if we want the tools and the app in the app bundle to both work, we need some way to have the libs exist in both Build/lib and Build/bin/App.app/Contents/lib. 

Simply copying the TARGET_FILE for each lib doesn't work, because that only copies the library, and not all its symlinks :S.

Reply:
I'm confused :laughing: 

I would expect something to happen like libraries are linked into Build/lib, then when the bundle is created, things are copied into Build/App.app/Contents/lib. That seems to be how it behaved before, albeit we had to do it ourselves - that broke just by moving the top-level cmake file up one directory?

Reply:
Using Ladybird as the top level CMake lists was always broken in this way. When using Lagom as top level we ended up with a different path for the apps that let them find the libs.... I think we didn't use install rpath in build or w/e

Reply:
I see. Unfortunate, but fine for now - I'm kinda surprised making bundles doesn't just work in CMake

Reply:
There's a things called Bundle utilities but as far as I can tell it kinda sucks and needs a lot of particular arguments... It also grabs wayyyy to many dependency libs